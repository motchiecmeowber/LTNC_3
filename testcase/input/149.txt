INSERT a number
INSERT b string
BEGIN
INSERT c number
INSERT d string
BEGIN
INSERT a string
INSERT b number
RPRINT
END
PRINT
END
RPRINT