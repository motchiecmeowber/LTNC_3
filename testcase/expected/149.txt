success
success
success
success
success
success
b//2 a//2 d//1 c//1
a//0 b//0 c//1 d//1
b//0 a//0